docs/Fixed Wing Landing.md

```diff
@@ -0,0 +1,100 @@
+# Fixed Wing Landing
+
+## Introducion
+
+INAV supports advanced automatic landings for fixed wing aircraft from version 7.1.
+The procedure is based on landings for man-carrying aircraft, so that safe landings at a specific location are possible.
+Supported are landings at safehome after "Return to Home" or at a defined LAND waypoint for missions. 
+
+## General procedure:
+
+1. After reaching Safehome/LAND Waypoint the altitude is corrected to "Approach Altitude".
+2. The aircraft circles for at least 30 seconds to determine the wind direction and strength.
+3. The landing direction and the approach waypoints are calculated on the basis of the measured wind parameters. If no headwind landing is possible or the wind strength is greater than "Max. tailwind" (see Global Parameters), return to point 2.
+4. The landing is initiated. The aircraft flies the downwind course, "Approach Altitude" is held.
+5. Base Leg: the altitude is reduced from 2/3 of "Approach Altitude".
+6. Final Appraoch: The engine power is reduced using "Pitch2throttle modifier" to reduce speed and the altitude is reduced to "Land Altitude".
+7. Glide: When "Glide Altitude" is reached, the motor is switched off and the pitch angle of "Glide Pitch" is held.
+7. Flare: Only if a LIDAR/Rangefinder sensor is present: the motor remains switched off and the pitch angle of "Flare Pitch" is held
+8. Landing: As soon as INAV has detected the landing, it is automatically disarmed, see setting `nav_disarm_on_landing`.
+
+To activate the automatic landing, the parameter `nav_rth_allow_landing` must be set to `ALWAYS` or `FAILSAFE`. 
+
+> [!WARNING]
+> If landing is activated and no parameters are set for the landing site (Safehome and/or landing waypoint), the old landing procedure (circling until close to the ground, then hovering out) is performed. 
+> This is probably not what you want. 
+ 
+The following graphics illustrate the process:
+
+![Approach Drawing Up](/docs/assets/images/Approach-Drawing-Up.png  "Approach Drawing Up")
+
+![Approach Drawing Side](/docs/assets/images/Approach-Drawing-Side.png  "Approach Drawing Side")
+
+## Landing site parameters
+
+### The following parameters are set for each landing site (Safefome/LAND waypoint):
+
+All settings can also be conveniently made in the Configurator via Missionplanner.
+
+CLI command `fwapproach`:
+`fwapproach <index> <Apprach direction> <Approach altitude> <Land altitude> <approach heading 1> <approach heading 2> <sea level>`
```

Comment:
Not consistent with the code, which implements `index`, `approach_alt`, `land_alt`, `land_direction` ...
Trivially `s/Apprach/Approach/` also.